tests/time_tests/test_runner/conftest.py

```diff
@@ -251,13 +307,13 @@ def pytest_runtest_makereport(item, call):
     """
 
     FIELDS_FOR_ID = ['timetest', 'model', 'device', 'niter', 'run_id']
-    FIELDS_FOR_SUBMIT = FIELDS_FOR_ID + ['_id', 'test_name',
-                                         'results', 'status', 'error_msg']
+    FIELDS_FOR_SUBMIT = FIELDS_FOR_ID + \
+                        ['_id', 'test_name', 'results', 'status', 'error_msg',
+                         'platform', 'repo_url', 'commit_sha',
+                         'commit_date', 'target_branch', 'ovino_version']
```

Comment:
Can we name it "version" instead "ovino_version".

Reply:
Done